tests/Microsoft.Identity.Test.Unit/CoreTests/InstanceTests/AadAuthorityTests.cs

```diff
@@ -333,6 +333,34 @@ public void IsOrganizationsTenantWithMsaPassthroughEnabled()
             Assert.IsTrue(aadAuthorityInstance.IsOrganizationsTenantWithMsaPassthroughEnabled(true, "9188040d-6c67-4c5b-b112-36a304b66dad"));
         }
 
+        [TestMethod]
+        public void IsCommonOrOrganizationsTenantTest()
```

Comment:
I'm surprised that we didn't have to update any unit tests. This indicates lack of code coverage.

Reply:
Oh yeah, I see, yeah these methods need more testing

Reply:
This test can be simplified taking input for common, organizations and consumers with an expected value true/false for aadAuthorityInstance.IsCommonOrOrganizationsTenant().

Reply:
I'm going to resolve this as it has been fixed. Thanks @trwalke . Pls create new comment threads as needed.